pages/common/php-artisan.md

```diff
@@ -0,0 +1,19 @@
+# php artisan
+
+> Laravel's Artisan command line interface.
+
+- Display a list of all available commands:
+
+`php artisan help`
```

Comment:
Wouldn't this get PHP to execute a file called `artisan` in the current directory?

I assume that a dash `-` was missed out here or something?

I'm not familiar with _Artisan_, so I could be wrong.

Reply:
Indeed, `artisan` is a file included with Laravel to be executed with PHP. I'm not entirely sure if the tldr project intends to cover things like this since it's not a global command, just one _very_ frequently used inside project directories by Laravel users.

I noticed there's some other command pages with spaces in (such as `git add`) but obviously, these are global. Is there any recommended way to handle this?

Reply:
Ah, I see. It's definitely something we should cover - though I'm unsure as to whether the page name should be `php artisan` or just simply `artisan`.

I think `php artisan` should either be `php ./artisan` or `./artisan` personally. @agnivade, what do you think?

Reply:
Perhaps one of these would be better, but some comments:

- `./artisan` would not be appropriate, as Windows wouldn't know what to execute the file as since it doesn't support the shebang.
- All the documentation, guides, tutorials, etc. around the framework refers to it as `php artisan` consistently, I'm fairly certain that anyone looking up the tl;dr of this command will understand that.
- If it is named `php ./artisan` would people still be able to access this page as `tldr php artisan`?

Reply:
If the official documentation calls it `php artisan`, then let's stick with that :-)